lib/membrane/video_compositor/video_compositor.ex

```diff
@@ -22,11 +22,37 @@ defmodule Membrane.VideoCompositor do
   @type update_transformations ::
           {:update_transformations, [{Membrane.Pad.ref_t(), VideoTransformations.t()}]}
 
+  @init_metadata_doc """
+  User-specified init metadata passed to handler callbacks.
+  Passing init metadata into `c:Membrane.VideoCompositor.Handler.handle_init/1` callback allows
+  the user to alternate custom-implemented init callback logic.
+  """
+
+  @typedoc @init_metadata_doc
```

Comment:
Why?

Reply:
Well, we use this attribute in two places. The first one is the `typedoc` and the second one is the `metadata option` `description`. 
I did it that way because: 
1. Having this description directly available in both places IMO makes it easier to read documentation than referencing the `typedoc` in the description.
2. Extracting `typedoc` in `description` using `#{inspect(Module.get_attribute/2)}` was painful. For me, it seemed to be an antipattern and over-engineering. 
3. Repeating this description twice obviously makes it harder to maintain and breaks DRY